.github/workflows/download-latest-tokenlist.yml

```
@@ -0,0 +1,22 @@
+        uses: actions/checkout@v4
+      - name: Fetch latest token list
+        run: |
+          curl -H "Accept: application/json" -X GET "https://production-api.mobula.io/api/1/all?fields=id,decimals,logo,contracts,blockchains" -o src/data/tokens.json
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          author_name: KioBot
+          author_email: kieran@pillarproject.io
+          message: 'KioBot downloaded the latest token list and updated tokens.json'
+          add: 'src/data/tokens.json'
+        
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)